### PR TITLE
Ft dsk 1562 monitor inprove

### PR DESCRIPTION
**Pull request by rafaeldelboni**

### CODE REVIEW

File: src/helper/db.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const low = require('lowdb')
+const FileSync = require('lowdb/adapters/FileSync')
+const adapter = new FileSync('db.json')
```
Comment (vdvleon) [inline src/helper/db.js:3]: Maybe smart to specify the path a little bit more clearly. I don't know if this is relative to this file, to the root of the project, etc. Maybe something like `path.resolve(__dirname, '../../db.json')`

**Review by vdvleon — APPROVED** [review]
Just to be sure, how does this check work? Is it only 'unhealthy' if it is unhealthy twice in a row?

Or does the following sequence also classify as unhealthy (where u=unhealthy, h=healthy): u, h, u

**Comment by rafaeldelboni** [comment]
It checks for the last 15 minutes executions if the same target had more than one error in this time gap it willl send a email. Don't need to be in a row.

**Comment by vdvleon** [comment]
@RafaelDelboni  a great, yeah that sounds good

**Pull request merged**